docs/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -38,176 +39,193 @@ As usual, this release includes important fixes, some of which may be critical f
+    - [example](https://github.com/ipfs/kubo/releases/tag/v0.17.0)
+  - [ ] ![](https://img.shields.io/badge/only-RC-blue?style=flat-square) create an issue comment mentioning early testers on the release issue
+    - [example](https://github.com/ipfs/kubo/issues/9319#issuecomment-1311002478)
+  - [ ] ![](https://img.shields.io/badge/only-FINAL-green?style=flat-square) create an issue comment linking to the release on the release issue
+    - [example](https://github.com/ipfs/kubo/issues/9417#issuecomment-1400740975)
+  - [ ] ![](https://img.shields.io/badge/only-FINAL-green?style=flat-square) ![](https://img.shields.io/badge/not-PATCH-yellow?style=flat-square) ask the marketing team to tweet about the release in [#shared-pl-marketing-requests](https://filecoinproject.slack.com/archives/C018EJ8LWH1) in FIL Slack
+    - [example](https://filecoinproject.slack.com/archives/C018EJ8LWH1/p1664885305374900)
+  - [ ] ![](https://img.shields.io/badge/only-FINAL-green?style=flat-square) ![](https://img.shields.io/badge/not-PATCH-yellow?style=flat-square) post the link to the [GitHub Release](https://github.com/ipfs/kubo/releases/tag/vX.Y.Z(-RCN)) to [Reddit](https://reddit.com/r/ipfs)
+    - [example](https://www.reddit.com/r/ipfs/comments/9x0q0k/kubo_v0160_release_is_out/)
+  </details>
+- [ ] Test the new version with `ipfs-companion` <details><summary>using `kuboreleaser release --version vX.Y.Z(-RCN) test-ipfs-companion` or ...</summary>
+  - [ ] run the [e2e](https://github.com/ipfs/ipfs-companion/actions/workflows/e2e.yml)
+    - use `vX.Y.Z(-RCN)` as the Kubo image version
+  - [ ] wait for the [e2e](https://github.com/ipfs/ipfs-companion/actions/workflows/e2e.yml) workflow run to finish
+  </details>
+- [ ] ![](https://img.shields.io/badge/only-FINAL-green?style=flat-square) Update Kubo in [interop](https://github.com/ipfs/interop) <details><summary>using `kuboreleaser release --version vX.Y.Z(-RCN) update-interop` or ...</summary>
+  - [ ] ![](https://img.shields.io/badge/only-FINAL-green?style=flat-square) check out [ipfs/interop](https://github.com/ipfs/interop)
+  - [ ] ![](https://img.shields.io/badge/only-FINAL-green?style=flat-square) run `npm install`
+  - [ ] ![](https://img.shields.io/badge/only-FINAL-green?style=flat-square) create a PR which updates `package.json` and `package-lock.json`
+  - [ ] ![](https://img.shields.io/badge/only-FINAL-green?style=flat-square) merge the PR
+  </details>
+- [ ] Update Kubo in [ipfs-desktop](https://github.com/ipfs/ipfs-desktop) <details><summary>using `kuboreleaser release --version vX.Y.Z(-RCN) update-ipfs-desktop` or ...</summary>
+  - [ ] check out [ipfs/ipfs-desktop](https://github.com/ipfs/ipfs-desktop)
+  - [ ] run `npm install`
+  - [ ] create a PR which updates `package.json` and `package-lock.json`
+  - [ ] ![](https://img.shields.io/badge/only-FINAL-green?style=flat-square) merge the PR
```

Comment:
Can we add a step here to ensure that @SgtPooki or @whizzzkid are tagged on this PR that is created? Also, can we add a note that unless you're performing the ipfs-desktop release, you should not merge that PR unless you want the ipfs-desktop maintainers to miss that we need to do a release :)

see https://github.com/ipfs/ipfs-desktop/issues/2407#issuecomment-1437430617

Reply:
```suggestion
- [ ] Update Kubo in [ipfs-desktop](https://github.com/ipfs/ipfs-desktop) <details><summary>using `kuboreleaser release --version vX.Y.Z(-RCN) update-ipfs-desktop` or ...</summary>
  - [ ] check out [ipfs/ipfs-desktop](https://github.com/ipfs/ipfs-desktop)
  - [ ] run `npm install`
  - [ ] create a PR which updates `package.json` and `package-lock.json`
  - [ ] ![](https://img.shields.io/badge/only-FINAL-green?style=flat-square) add @SgtPooki and @whizzzkid as reviewers
```